docs/community-toolkit/ollama.md

```diff
@@ -61,6 +61,16 @@ var ollama = builder.AddOllama("ollama")
                     .WithDataVolume();
 ```
 
+### Use GPUs when available
+
+One or more LLMs are downloaded into the container which Ollama is running from, and by default this container runs on CPU. If you need to run the container in GPU you need to pass a parameter to the container runtime args.
+
+```csharp
```

Comment:
We should update this to use the tab syntax, we actually have an example of Ollama GPUs here:

https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/app-host-overview?tabs=docker#express-container-resources

And this is the desired syntax:

https://github.com/dotnet/docs-aspire/blob/84d99db2af0fe21dadece048807c3f6dada0a6df/docs/fundamentals/app-host-overview.md#L110-L140